kge/model/embedder/lookup_embedder.py

```diff
@@ -107,21 +107,31 @@ def _embeddings_all(self) -> Tensor:
     def _get_regularize_weight(self) -> Tensor:
         return self.get_option("regularize_weight")
 
+    def normalize_complex(self, parameters) -> Tensor:
```

Comment:
This is not normalizing, but computing the element-wise absolute value. I suggest to rename to _abs_complex.

Reply:
changed ^^ thank for suggestion.